docs/developers/deploy-on-polygon.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Deploying the LSP on Polygon
+sidebar_label: Deploying the LSP on Polygon
+---
+UMA's infrastructure is also available on Polygon for developers to make use of as a scaling solution. The following will explain the details on how to deploy on Polygon. 
+
+Deploying to Polygon follows a near-identical launch process as to launching any UMA contract on Ethereum. Using one of our launch repo's and amending the destination network, you can quickly deploy your contracts on Polygon.
+
+## How UMA works on Polygon
+
+UMA makes use of an arbitrary message bridge that allows for two-way messages to pass between the Polygon network and Ethereum mainet. The message bridge makes use of the Polygon Arbitrary Message system, meaning it can be as trusted as the Polygon network itself. 
+
+UMA's Optimistic Oracle is used as the arbiter of price requests locally on the Polygon network. A request will ask the locally deployed Optimitic Oracle. If the proposal goes undisputed, then the result is deemed the accepted outcome. 
+
+If the event is disputed, then the following steps will be taken to bridge the disputed result back to the final arbitator, the DVM:
+1. A Polygon contract, such as a prediction market, needs a price to settle a payout. The contract expects to get this price from an optimistic oracle (“Polygon Oracle”).
+2. For some reason, a user disagrees with the price returned by the Polygon Oracle and disputes the price.
+3. The disputed price request is passed from the Polygon Oracle to a contract called the “Oracle Child Tunnel”, whose sole responsibility is to communicate with an “Oracle Root 
+4. Tunnel” on the Ethereum network. The Child Tunnel relays the dispute to Ethereum mainnet to the Root Tunnel.
+5. The Oracle Root Tunnel has special permission to request a price from the DVM, where the familiar voting and resolution process occurs amongst UMA voting token holders.
+Once the DVM has resolved a price request, the outcome of the vote is pushed to Oracle Root Tunnel. It is important to note that the DVM is not aware of which chain the request came from, nor does it need to.
+6. Like before, the Oracle Root Tunnel relays the result from the DVM to the Child Tunnel on Polygon.
+7. Finally, the Oracle Child Tunnel then sends a message back to the Polygon Oracle
+8. The outcome of the dispute is resolved.
+
+![](/img/PolygonDispute.png)
+
+## Deployment process
```

Comment:
Do you think it is worth adding a "Check to make sure relevant price ids and collateral types are approved" steps? Like we have in the [launch-emp tutorial](https://docs-git-polygon-deplyment-uma.vercel.app/developers/emp-deployment#is-your-desired-price-identifier-and-collateral-type-supported)?

Reply:
Yea can do. Think we need to actually find a way to represent the subset of collateral addresses that are Polygon compatible or whitelisted. Adding this as a sub-task for me. 

I added just a boiler plate answer to this

Reply:
Looks good!